src/mission.h

```diff
@@ -371,6 +374,7 @@ class mission
         int get_id() const;
         const std::string &get_item_id() const;
         character_id get_npc_id() const;
+        std::vector<std::pair<int, std::string>> get_likely_rewards();
```

Comment:
```suggestion
        const std::vector<std::pair<int, std::string>> &get_likely_rewards() const;
```

Same for the function in the other class.